piximi-documentation/measurements.md

```diff
@@ -4,4 +4,104 @@ While sometimes, the goal of an experiment is just to find objects or to classif
+
+```{figure} ./img/measurement-walkthrough/measure-00-pre-measurement.png
+---
+name: measurement-00-table-plus
+scale: 50%
+---
+The `+` button at the top of the Tables menu allows you to add a new table
+```
+
+3. Select a `kind` you'd like to make measurements of. Reminder, `kind`s include the images, as well as any objects you created via segmentation or annotation.
+
+```{figure} ./img/measurement-walkthrough/measure-01-create-table.png
+---
+name: measure-01-create-table
+scale: 50%
+---
+Select what kind of measurement table you'd like to make
+```
+
+4. Piximi will begin to calculate metrics in the background. Depending on the number of examples present in that `kind`, this may take a few minutes - an indicator will show you the progress made.
+
+```{figure} ./img/measurement-walkthrough/measure-02-spin-while-creating-table.png
+---
+name: measure-02-spin-while-creating-table
+scale: 50%
+---
+The circular indicator next to `Tables` indicates how far through measurement generation Piximi is. Make sure to keep the Piximi window open (consider moving it into its own tab) to let calculations continue!
+```
+
+5. Once the initial pre-calculations are done, you can select which measurements you'd like to make, as well as how you'd like to subset "split" the data (currently by `Category` or by `Partition`; if you only have one category, tha's fine, but you do have to click something here!). As before, this will take some time, so just leave the Piximi tab open and the circular indicator will let you know how long this is going to take.
+
+```{figure} ./img/measurement-walkthrough/measure-03-select-splits-and-measurements.png
+---
+name: measure-03-select-splits-and-measurements
+scale: 50%
+---
+Select the split or splits you want to measure for each table, as well as the measurements you want for that table. Just like during table creation, a circular indicator next to the word `Measurements` will tell you how far along Piximi is at generating your measurements.
+```
+
+6. Once your measurements have been generated, they will appear in the main area in a data grid! In the grid, you can see the measurement name, the subset (or `Split`) of the data represented, and the Mean, Median, and Standard deviation of the data for that feature in that subset.
+
+```{figure} ./img/measurement-walkthrough/measure-04-data-grid.png
+---
+name: measure-04-data-grid
+---
+The Piximi data grid
+```
+
+7. If you'd like to plot data in Piximi, simply navigate to the data grid containing the data you want to plot and hit the Plots tab at the top. You can make as many plots as you would like by hitting the `+` button on the plots tab. Plots can also be exported by saving to PNG. Piximi includes several plot types, including scatter plots, swarm plots, and histograms.
+
+```{figure} ./img/measurement-walkthrough/measure-05-scatter.png
+---
+name: measure-05-scatter
```

Comment:
I don't love that for our docs example the color palette only has 6 colors but there are more than 6 categories selected in the example so multiple points have the same color. I can let it go if the goal is to quickly get some functional docs merged in. Maybe could be made into an issue for low-hanging future fruit?

Reply:
I think Andrea wants to eventually switch plotting libraries entirely (and switch the docs when he does), but we do want something in for when we send out for review, so we should probably live with it for now.